src/crypto/index.ts

```diff
@@ -990,11 +999,13 @@ export class Crypto extends EventEmitter {
         return this.secretStorage.get(name);
     }
 
-    public isSecretStored(name: string, checkKey?: boolean): any { // TODO types
+    public isSecretStored(
+        name: string, checkKey?: boolean,
+    ): Promise<Record<string, ISecretStorageKeyInfo>> {
         return this.secretStorage.isStored(name, checkKey);
     }
 
-    public requestSecret(name: string, devices: string[]): Promise<any> { // TODO types
+    public requestSecret(name: string, devices: string[]): ISecretRequest { // TODO types
```

Comment:
```suggestion
    public requestSecret(name: string, devices: string[]): Promise<ISecretRequest> {
```

Reply:
Actually looking at `setDefaultKeyId` it should be `Promise<void>`?

Reply:
I think this is right - it returns an object with the promise and also a cancel function and a request ID

Reply:
oh I think the diff was cut and I was the tail end of a different method - you are correct.